<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C21 ログ解析</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QPushButton" name="pushOutput">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>書き出し</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>ログ解析</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>遺跡</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/dungeon.png</normaloff>:/res/dungeon.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>撃破</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/battle.png</normaloff>:/res/battle.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>取得/アイテム</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/item.png</normaloff>:/res/item.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>取得/パーツ</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/parts.png</normaloff>:/res/parts.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>取得/ギルドポイント</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/other.png</normaloff>:/res/other.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>報酬/アイテム・パーツ</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/item.png</normaloff>:/res/item.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ガチャ</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/gatya.png</normaloff>:/res/gatya.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>基地</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/mission_1.png</normaloff>:/res/mission_1.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>突発</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/mission_2.png</normaloff>:/res/mission_2.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ラボ</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/res/labo.png</normaloff>:/res/labo.png</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushCount">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>カウント</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QTreeWidget" name="treeResult">
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <property name="indentation">
           <number>0</number>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>周回数/撃破数/獲得数</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>ログ検索</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkUser">
            <property name="text">
             <string>ユーザー発言をスキップ（NPC含む）</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkInfo">
            <property name="text">
             <string>［INFO］ログをスキップ</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelFile">
            <property name="text">
             <string>対象ファイル数　：　</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelText">
            <property name="text">
             <string>検索文字列　：　</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSearch">
            <property name="text">
             <string>合致数　：　</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QTreeWidget" name="treeSearch">
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <property name="indentation">
           <number>0</number>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>日時</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ログ</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineSearch"/>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushSearch">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>検索</string>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/res/search.png</normaloff>:/res/search.png</iconset>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QFrame" name="frameOption">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="3">
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>指定した期間</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">radioGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>今日</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">radioGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>指定した日</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">radioGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>すべて</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">radioGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,1">
         <item>
          <widget class="QComboBox" name="comboStart"/>
         </item>
         <item>
          <widget class="QLabel" name="labelOption">
           <property name="text">
            <string>から</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboEnd"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushCapture">
        <property name="text">
         <string>画像書き出し</string>
        </property>
        <property name="icon">
         <iconset resource="../res.qrc">
          <normaloff>:/res/photo.png</normaloff>:/res/photo.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushOption">
        <property name="text">
         <string>設定</string>
        </property>
        <property name="icon">
         <iconset resource="../res.qrc">
          <normaloff>:/res/tools.png</normaloff>:/res/tools.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>ファイル(&amp;F)</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="HelpMenu">
    <property name="title">
     <string>ヘルプ(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="HelpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/quit.png</normaloff>:/res/quit.png</iconset>
   </property>
   <property name="text">
    <string>終了</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/help.png</normaloff>:/res/help.png</iconset>
   </property>
   <property name="text">
    <string>ヘルプ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/page.png</normaloff>:/res/page.png</iconset>
   </property>
   <property name="text">
    <string>このソフトについて</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/page.png</normaloff>:/res/page.png</iconset>
   </property>
   <property name="text">
    <string>コピー</string>
   </property>
   <property name="toolTip">
    <string>コピー</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushSearch</sender>
   <signal>clicked()</signal>
   <receiver>treeSearch</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="radioGroup"/>
 </buttongroups>
</ui>
